recipe/meta.yaml

```diff
@@ -1,42 +1,54 @@
 {% set name = "DoubleMetaphone" %}
 {% set version = "1.1" %}
-{% set md5 = "eb6a3f05f3254db7268d462cd0a3ff49" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  md5: {{ md5 }}
+  # Use upstream archive tarball for tests
+  url: https://github.com/dedupeio/doublemetaphone/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 69eb748a08e8d7d7f4448e8bfbc9b5d5018d979067c9b48c62d121633e4f7b55
 
 build:
-  number: 2
-  script: python setup.py install --single-version-externally-managed --record record.txt
-  skip: true  # [win and py3k]
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
   host:
     - python
+    - pip
     - setuptools
+    - wheel
     - cython
-
   run:
     - python
 
 test:
+  source_files:
+    - tests
   imports:
     - doublemetaphone
+  requires:
+    - pip
+    - pytest
+  commands:
+    - pip check
+    - pytest -v tests
 
 about:
-  license: MIT
+  home: https://github.com/datamade/doublemetaphone
+  license: Artistic-2.0
   license_file: LICENSE
+  license_family: PSF
```

Comment:
I'm not sure this is right. Artistic-2.0 seems to be a Perl license; PSF stands for Python Software Foundation. AFAIU. Maybe "other"? https://github.com/conda/conda-build/blob/ee7d0cea9831565ee3cef19708bad05ae930b2d5/conda_build/license_family.py#L10

Reply:
@skupr-anaconda what do you think

Reply:
`Other` is fine